cdisc_rules_engine/utilities/sql_rule_processor.py

```diff
@@ -91,6 +103,21 @@ def rule_applies_to_domain(cls, dataset_metadata: SQLDatasetMetadata, rule: dict
 
         return True
 
+    @staticmethod
+    def _is_split_dataset(domain_name: str) -> bool:
```

Comment:
These should be decided when the dataset metadata is created. Why would they be done now?